cmd/soroban-cli/Cargo.toml

```diff
@@ -14,7 +14,7 @@ default-run = "soroban"
 
 [[bin]]
 name = "stellar"
-path = "src/bin/main.rs"
+path = "../stellar-cli/src/bin/main.rs"
```

Comment:
Not sure if it's the cleanest way, but this way we are ensuring that we are pointing to a correct binary (doubt we would want to change soroban-cli and stellar-cli binaries in the future)

Reply:
We should ignore this warning. It's fine to have two binaries reference the same main. The warning is just to catch people me doing a mistake. We are doing it intentionally.

Reply:
There should be an option we can add to the projects `.cargo/config.toml` file or the the GitHub ci workflow to disable that warning.

Reply:
Should we maybe just copy main to silence this warning?

Reply:
There's only `RUSTFLAGS=-Awarnings` which silences all warnings, which is not ideal. I'd rather either copy file (done in the last commit), or just keep the warning there

Reply:
> Should we maybe just copy main to silence this warning?

Yeah, that would be preferred over referencing the other crate file which is technically outside the project.

The files are small, couple copies is fine.